### ISSUE TITLE
the .gitignore isn't getting written...

### ISSUE THREAD

**Issue by the-simian** [the-simian/slush-phaser-webpack#8]
after you template a project, its just not there.

**Comment by rblopes** [comment]
Yup. You already have the routine to rename dotfiles in your `default` task, but you are missing the files to rename :smiley:

So, under your default project template, you should rename the `.gitignore` to read `_gitignore`, and that's it.

**Comment by the-simian** [comment]
Yeah, tracked it down to that last night. I'll probably update it this evening if I can.
Thanks for posting!
